script/Arcueid/Day 10/10_00_ARC10_7_2B.txt

```diff
@@ -281,12 +297,12 @@ Stop. Don't go. Turn back.
 [sha:874d12e068cae8845b2b0b5fa297ecf3c8954e7f]{
 -- Page 788, Offset 26442.
 -- この先は、おまえが夢見ているようなモノではない、と。
-What lies ahead is not part of a dream.
+What lies ahead is not what you've been dreaming of.
```

Comment:
I think this might be worth reverting. "what you've been dreaming of" in english makes it seem like "something you've been aspiring to" when this is more  like phantasmagoria